draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -43,75 +43,37 @@ organization="Ping Identity"
-the Verifier becomes crucial to ensure minimum disclosure and prevent
-Verifiers from obtaining claims irrelevant for the transaction at hand.
-SD-JWTs defined in this document enable such selective disclosure of JWT claims.
-
-One example of a multi-use JWT is an Issuer-signed
-credential that contains the claims about a subject, and whose authenticity can be
-cryptographically verified.
-
-Similar to the JWT specification on which it builds, this document is a product of the
-Web Authorization Protocol (OAuth) working group. However, while both JWT and SD-JWT
-have potential OAuth 2.0 applications, their utility and application is certainly not constrained to OAuth 2.0.
-JWT was developed as a general-purpose token format and has seen widespread usage in a
-variety of applications. SD-JWT is a selective disclosure mechanism for JWT and is
-similarly intended to be a general-purpose specification.
-
-While JWTs with claims describing natural persons are a common use case, the
-mechanisms defined in this document are also applicable to other use cases.
-
-In an SD-JWT, claims can be hidden, but cryptographically
-protected against undetected modification. When issuing the SD-JWT to
-the Holder, the Issuer includes the cleartext counterparts of all hidden
-claims, the so-called Disclosures, outside the signed part of the SD-JWT.
-
-The Holder decides which claims to disclose to a particular Verifier and includes the respective
-Disclosures in the SD-JWT to that Verifier. The Verifier
-has to verify that all disclosed claim values were part of the
-Issuer-signed JWT. The Verifier will not, however, learn any hidden claim
-values not disclosed in the Disclosures.
-
-This document also defines a format for SD-JWTs with Key Binding (SD-JWT+KB).
-By optionally sending an SD-JWT+KB to a
-Verifier, the Holder can prove to the Verifier that they hold the private key
-associated to the SD-JWT (i.e., using the `cnf` claim [@!RFC7800]). The strength of the binding is conditional upon the trust
-in the protection of the private key of the key pair an SD-JWT is bound to.
-
-SD-JWT can be used with any JSON-based representation of claims.
-
-This specification aims to be easy to implement and to leverage
-established and widely used data formats and cryptographic algorithms
-wherever possible.
+JSON-encoded data structures for exchange between systems are often secured against modification using JSON Web Signatures (JWS) [@!RFC7515].
+A popular application of JWS are JSON Web Tokens (JWT) [@!RFC7519], which are often used to represent a user's identity.
+An ID Token as defined in OpenID Connect [@?OpenID.Core], for example, is a JWT containing the user's claims created by the server for consumption by a client.
+In case the JWS is sent directly from the server to the client, as in OpenID Connect, the server can select which user claims to include in the JWT, minimizing the information shared with the client.
```

Comment:
Isn't the important fact here that the client is also the party verifying the JWT? I think, adding this detail may be useful to understand the issuer-holder-verifier model defined afterwards as it may otherwise be hard for the reader to understand the fundamental shift.

Reply:
```suggestion
In case a JWT is sent directly from a server to a client, as in OpenID Connect, the server can select real-time which user claims to include in the JWT, minimizing the information shared with the client, who validates the JWT.
```

Reply:
reflecting (maybe) prior comments along with my own nitpicks to create this frankenstein of a sentence : 
```suggestion
In cases where the JWT is sent immediately from the server to the relying party, as in OpenID Connect, the server can select at the time of issuance which user claims to include in the JWT, minimizing the information shared with the relying party, who validates the JWT.
```

Reply:
```suggestion
In cases where the JWT is sent immediately from the server to the relying party, as in OpenID Connect, the server can select at the time of issuance which user claims to include in the JWT, minimizing the information shared with the relying party who validates the JWT.
```